config/initializers/jwt_session.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+JWTSessions.algorithm = 'HS256'
+JWTSessions.encryption_key = Rails.application.credentials.secret_key_base
+JWTSessions.token_store = :redis, {
+    redis_host: '127.0.0.1',
```

Comment:
Why???? What about use memory store for tests, otherwise use REDIS_DB?
Something like:
```ruby
JWTSessions.token_store = Rails.env.test? ? :memory : [:redis, { redis_url: ENV['REDIS_DB'] }]
```

Reply:
Done